src/parser/extract-id.test.js

```diff
@@ -0,0 +1,7 @@
+import { extractId } from './extract-id';
+
+describe('extract-id', () => {
+  test('name with spaces', () => {
+    expect(extractId({ name: 'Name with spaces' })).toBe('Name_with_spaces');
```

Comment:
Is underscore the convention or just preference?

Reply:
A preference, I have to generate a valid js id from a story name.

Reply:
I wonder if `NameWithSpaces` would be the more common way to write a story ID